client/components/widget/query/query-editor-controller.js

```diff
@@ -196,8 +185,9 @@ explorer.components.widget.query.QueryEditorCtrl = function($scope, $filter,
 
   $scope.$watch(
       angular.bind(this, function() {
-        if (this.datasource) {
-          return this.datasource.config.filters.test;
+        if (dashboardService.selectedWidget) {
+          return dashboardService.selectedWidget.model
```

Comment:
This class will be deleted on the next PR (in favor of sidebar).  Will address improvements in new classes.